services/cfg.py

```diff
@@ -1,5 +1,33 @@
+from enum import Enum
 
 
-
-ARTIST_INDEX = 1
-LIKES_INDEX = 22
\ No newline at end of file
+class TableEnum(Enum):
```

Comment:
It is a Enum that stores tables indexes, which is not represented in a "TableEnum" class name.